pom.xml

```diff
@@ -657,8 +657,8 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <carbon.kernel.version>4.4.1</carbon.kernel.version>
-        <carbon.social.version>2.3.5</carbon.social.version>
-        <carbon.store.version>2.3.5</carbon.store.version>
+        <carbon.social.version>2.3.5-SNAPSHOT</carbon.social.version>
+        <carbon.store.version>2.3.5-SNAPSHOT</carbon.store.version>
```

Comment:
This is the master branch of product ES which have carbon social and carbon store as its native components. So what else we should use other than snapshot versions of the components for snapshot version of the product?